Corona-Warn-App/src/main/assets/terms_tr.html

```diff
@@ -59,797 +58,774 @@
     13353 Berlin ("<strong>RKI</strong>")
 </p>
 <p>
-    vertreten durch den Präsidenten. Für Fragen, Beschwerden und Ansprüche im
-    Zusammenhang mit diesen Nutzungsbedingungen können Sie das RKI telefonisch
-    unter +49 30 18754-5100 und per E-Mail unter CoronaWarnApp@rki.de
-    erreichen.
+    Başkanı tarafından temsil edilir. Bu Kullanım Koşulları ile ilgili
+    sorularınız, şikayetleriniz ve talepleriniz için, +49 30 18754-5100
```

Comment:
According to EXPOSUREAPP-2633 we should remove this Hotline. Also in EN. I've done it for DE.

Reply:
EN Text:
"which is represented by its President. Please email CoronaWarnApp@rki.de if you have any 
questions, complaints or legal claims relating to these Terms of Use."